discrip.py

```diff
@@ -1,213 +1,6 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python
```

Comment:
Should keep `python3` as Ubuntu 24.04 does not recognize `python`. All other files in this repo use `python3`.